nodejs/exceljs/getConnectionString.js

```diff
@@ -0,0 +1,22 @@
+require('dotenv').config();
+
+function getConnectionString() {
```

Comment:
Some suggested changes to this method (and the corresponding README):

- We should also have a `DB_DSN`
- We should only add `UID=` and `PWD=` to the connection string if those values actually exist
- We should default to `DSN=*LOCAL` if nothing else is specified

Reply:
I still think this needs some tweaking, since someone could specify a DSN as well as a UID/PWD. 
Maybe something like:
```javascript
    let connectionString = 'DSN=*LOCAL;';
    if (process.env.DB_DSN) { // using DSN to connect
      connectionString = `DSN=${process.env.DB_DSN};`
    }

    if (process.env.DB_USER) { // using uid/password to connect
      connectionString = `DRIVER=IBM i Access ODBC Driver;UID=${process.env.DB_USER};`;

      if (!process.env.DB_PASS) {
        throw Error('"DB_PASS" environment variable must be defined');
      }

      connectionString += `PWD=${process.env.DB_PASS};`;
      connectionString += process.env.DB_HOST ? `SYSTEM=${process.env.DB_HOST};`
                                              : `SYSTEM=${localhost};`;
    } 
```

Reply:
That is true. Will adjust the script accordingly.